edit_and_sign.py

```diff
@@ -197,30 +197,89 @@ def edit_profile(merged_profile, choices_file, cli_mode):
 
 
 def cli_choices(choices_file, email, repos, skills):
+    RELEVANT = 'Relevant'
```

Comment:
There are constants already. We should use the same so that behavior is same in UI and CLI

Reply:
removed locally defined constants